pip_audit/_dependency_source/requirement.py

```diff
@@ -89,6 +89,18 @@ def collect(self) -> Iterator[Dependency]:
                 reqs: List[InstallRequirement] = []
                 req_names: Set[str] = set()
                 for req in rf.requirements:
+                    if req.req is None:
+                        # For editable or VCS requirements that don't have an egg fragment that
```

Comment:
I believe this behaviour is identical to what `pip-api` was doing. Without an egg fragment, it used to just return the actual requirement line as the package name.